src/Checker/AdminChecker.php

```diff
@@ -47,33 +42,15 @@ public function isMaintenance(MaintenanceConfiguration $configuration, Request $
         $adminPrefix = $this->params->get('sylius_admin.path_name');
 
         if (false !== mb_strpos($getRequestUri, $adminPrefix, 1)) {
-            if ($this->getMainRequest() === $this->requestStack->getCurrentRequest()) {
-                $this->flashBag->add('info', $this->translator->trans('maintenance.ui.message_info_admin'));
+            if ($this->requestStack->getMainRequest() === $this->requestStack->getCurrentRequest()) {
+                /** @var FlashBagInterface $flashBag */
+                $flashBag = $this->requestStack->getSession()->getBag('flashes');
```

Comment:
why here we call `getBag('flashes')` and elsewhere, we call `getFlashBag()` ?

Reply:
Because Interface isn't update and she haven't `getFlashBag()` ...
I do that for fix PHPStan alerte ... yes, it's not the best way :(